CMakeLists.txt

```diff
@@ -43,6 +61,19 @@ set(run_unittests OFF)
 set(run_traceability OFF)
 set(run_reals_check OFF)
 
+if ((WIN32) AND ("${GBALLOC_LL_TYPE}" STREQUAL "JEMALLOC"))
+    if (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/vcpkg/scripts/buildsystems/vcpkg.cmake)
+        set(VCPKG_OVERLAY_TRIPLETS "${CMAKE_CURRENT_LIST_DIR}/triplets")
+        if(${fsanitize})
+            set(VCPKG_TARGET_TRIPLET "x64-windows-static-comwrapper-asan")
+        else()
+            set(VCPKG_TARGET_TRIPLET "x64-windows-static-comwrapper")
+        endif()
```

Comment:
I think all this triplet stuff would be nice in c-build-tools. This whole block could be a function/macro from c-build-tools too so we don't have to do it in all the repos. But it sounds like moving things over there is the eventual plan anyway...